custom_components/tuya_local/devices/solar_inverter_stepuptech.yaml

```diff
@@ -0,0 +1,135 @@
+name: Micro Inverter Stepuptech SP800
+products:
+  - id: r73jxzrtcda4uwwb
+    name: SP800
+primary_entity:
+  entity: sensor
+  name: Daily energy out
+  class: energy
+  dps:
+    - id: 107
+      type: integer
+      name: sensor
+      unit: kWh
+      class: total_increasing
+      force: true
```

Comment:
Do all of these need to be forced?
Usually this is only necessary for some smartplugs, dedicated meters usually give frequent updates automatically, as this is their primary function,

Reply:
No, it's a mistake